test/e2e-common.sh

```diff
@@ -159,6 +167,13 @@ function install_knative_serving_standard() {
   wait_until_pods_running knative-serving || return 1
   wait_until_pods_running istio-system || return 1
   wait_until_service_has_external_ip istio-system istio-ingressgateway
+
+  if [[ -n "${INSTALL_MONITORING}" && -n "${INSTALL_MONITORING_YAML}" ]]; then
```

Comment:
this should probably be OR. I can ask to install monitoring bundle with serving or I should pass the flag to install it as needed

Reply:
Updated.